### DISCUSSION TITLE
Enterohepatic Recirculation

### DISCUSSION THREAD

**Discussion by pburkat1** [Open-Systems-Pharmacology/Forum#1549] [Questions & Problems]
Good morning, where can I find information on incorporating enterohepatic recirculation in PK-Sim simulations?

**Comment by tobiasK2001** [comment]
Good morning,

to include enterohepatic recirculation in PK-Sim simulations the most common way is to include events: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-events#definition-of-new-events-in-pk-sim-r
Best, Tobias

**Comment by Yuri05** [comment]
Additional/alternative to "discrete" EHC via gallbladder emptying created by an event as shown by @tobiasK2001 above: 
you can introduce **continuous** EHC by setting `EHC Continuous fraction` > 0 
Make sure to activate the advanced mode to see the parameters. 
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/c6e507e2-8361-4f44-a355-1928f0fa6af2)

**Reply by StephanSchaller** [reply]
And donÄt forget to ad Billiary Clearance or a transporter... otherwise there will be nothing in bile...
